js/setupFirebase.js

```diff
@@ -0,0 +1,33 @@
+const firebaseConfig = {
```

Comment:
this pull request is about video blocks please remove firebase related content

Reply:
All removed now.